### ISSUE TITLE
questions about settings in Table 6

### ISSUE THREAD

**Issue by squirrel16** [fanq15/Few-Shot-Object-Detection-Dataset#17]
In Table 6 (Experimental results on FSOD test set for 200 novel categories with 5 supports evaluated in novel category detection.), 
1. Did the first row mean Faster R-CNN pre-trained on ImageNet [56] and MS COCO [13], but not including FSOD? Then you fine-tune this model on the 200-way-5-shot support set?
2. Did the second row mean Faster R-CNN pre-trained on ImageNet [56] and MS COCO [13] and FSOD? Then you fine-tune this model on the 200-way-5-shot support set?
3. In the last row, what datasets did this model (your method) pre-trained on? what the difference between "trained directly" and the "❌ ✅" in the second column？

Thanks a lot for your reply!

**Comment by fanq15** [comment]
The default pretraining is on ImageNet and MS COCO.
1. Yes.
2. Yes. But we do not finetune on the full support set directly. We finetune and test the model separately on each category because we want to simulate the real application scenario where there are usually only a few novel categories at a time (how can we get 200 novel categories at a time?).
3. We train our model on the FSOD train set. Because it is trained on the FSOD and no fine-tuning, so it is not accurate to directly mark ✅in this row. The second column only represents fine-tuning on the FSOD test set.

**Comment by squirrel16** [comment]
Thanks very much!
